scripts/python/archive_repositories.py

```diff
@@ -60,33 +50,51 @@ def ready_for_archiving(repository, archive_date) -> bool:
     return False
 
 
-# How long ago in which the repositories should be archived
-archive_date_days = 0
-archive_date_months = 6
-archive_date_years = 1
-
-archive_date = datetime.now() - relativedelta(
-    days=archive_date_days, months=archive_date_months, years=archive_date_years
-)
-
-# Create MoJGithub object
-moj_gh = MojGithub(org=organization, org_token=org_token)
-
-# Get all repos that need archiving
-repos = [
-    repo
-    for repo in moj_gh.get_unarchived_repos("all")
-    if ready_for_archiving(repo, archive_date)
-]
-
-# Print repos
-logging.info(
-    f"Beginning archive of inactive repositories for GitHub organization: {organization}"
-)
-logging.info("-----------------------------")
-logging.info(f"Searching for inactive repositories from date: {archive_date}")
-logging.info("-----------------------------")
-
-# Archive repos
-for repo in repos:
-    MojArchive(repo).archive()
+def main():
+    # Config
+    # Logging Config
+    logging.basicConfig(
```

Comment:
Sonarlint wants this config reviewed - I'll centralise the logging configuration between scripts at a later date and verify then. For now, this config has just been lifted and shifted.